### PR TITLE
2.2 duplicate queries

### PR DESCRIPTION
**Pull request by dereuromark**
A small improvement when trying to find duplicate queries, they will be marked in red.

See img:
unbenannt

### CODE REVIEW

File: View/Elements/sql_log_panel.ctp [hunk 1]
```diff
@@ -41,6 +41,19 @@ if (isset($debugKitInHistoryMode)) {
...
+				$duplicate = 0;
+				foreach ($queryLog['queries'] as $k => $v) {
+					$hash = sha1($v['query']);
+					if (!isset($hashes[$hash]) || $hashes[$hash] !== $v['affected']) {
+						$hashes[$hash] = $v['affected'];
+						continue;
+					}
+					$duplicate++;
+
+					$queryLog['queries'][$k]['query'] = '<span style="color: red"">' . $v['query'] . '</span>';
```
Comment (markstory) [inline View/Elements/sql_log_panel.ctp]: You could use a CSS class instead of inline CSS.

Comment (dereuromark) [inline View/Elements/sql_log_panel.ctp]: I agree. Was just gonna wait for more feedback before giving it a finalize touch.

Comment (markstory) [inline View/Elements/sql_log_panel.ctp]: Ok.

File: View/Elements/sql_log_panel.ctp [hunk 2]
```diff
@@ -41,6 +41,19 @@ if (isset($debugKitInHistoryMode)) {
 					echo ' ' . __d('debug_kit', 'No query logs.');
 				endif;
 			else:
+				$hashes = array();
+				$duplicate = 0;
+				foreach ($queryLog['queries'] as $k => $v) {
```
Comment (ravage84) [inline View/Elements/sql_log_panel.ctp]: ``$key`` and ``$value`` or at least ``$val`` please!

**Comment by dereuromark** [comment]
Refactored as per comments.

**Pull request merged**